tensorflow/core/common_runtime/mkl_cpu_allocator.h

```diff
@@ -113,6 +96,7 @@ class MklSmallSizeAllocator : public Allocator {
   // Increment statistics for the allocator handling small allocations.
   inline void IncrementStats(size_t alloc_size)
       EXCLUSIVE_LOCKS_REQUIRED(mutex_) {
```

Comment:
Since you acquire the lock in the function,
```suggestion
      LOCKS_EXCLUDED(mutex_) {
```
Please also run `clang-format` as this might fit in the previous line.